datastruct/seqptrmap.c

```diff
@@ -74,7 +74,7 @@ seqptrmap_add(struct seqptrmap * M, void * ptr)
 void *
 seqptrmap_get(struct seqptrmap * M, int64_t i)
 {
-	size_t pos = i - M->offset;
+	size_t pos = (size_t)(i - M->offset);
```

Comment:
Hmm, this code is actually buggy with a 32-bit size_t... if `i` is 2^32 away from a valid value, we could return a pointer when we should have returned NULL.